src/coreclr/inc/corinfo.h

```diff
@@ -819,7 +819,6 @@ enum CORINFO_EH_CLAUSE_FLAGS
     CORINFO_EH_CLAUSE_FILTER    = 0x0001, // If this bit is on, then this EH entry is for a filter
     CORINFO_EH_CLAUSE_FINALLY   = 0x0002, // This clause is a finally clause
     CORINFO_EH_CLAUSE_FAULT     = 0x0004, // This clause is a fault clause
-    CORINFO_EH_CLAUSE_DUPLICATE = 0x0008, // Duplicated clause. This clause was duplicated to a funclet which was pulled out of line
     CORINFO_EH_CLAUSE_SAMETRY   = 0x0010, // This clause covers same try block as the previous one
```

Comment:
```suggestion
    CORINFO_EH_CLAUSE_SAMETRY   = 0x0008, // This clause covers same try block as the previous one
```
?

Reply:
Not sure I want to renumber it. There's at least one place in VM which uses the numeric value... and at least two different enums covering the same set of values.

Reply:
`corinfo.h` is JIT/EE interface. It should be fine to renumber it here as long as you rev JIT/EE interface GUID.

Reply:
I'd rather keep the enums in sync unless there's a specific need not to do so or performance benefit.

Reply:
Ok, it sounds like this is one of those places where JIT/EE interface definition leaks into the file formats and throughout the runtime. It is fine to keep it as is.

Reply:
It would be nice to add a comment like "Keep values in sync with COR_ILEXCEPTION_CLAUSE flags" to point the coupling.

Reply:
Added the comment.